test/builder_test.js

```diff
@@ -262,7 +262,8 @@ test('Builder', function (t) {
             name: 'parent',
             broccoliNode: true,
             broccoliId: 0,
-            broccoliCachedNode: false
+            broccoliCachedNode: false,
+            broccoliPluginName: undefined
```

Comment:
These are `undefined` because for these particular tests, we just use a POJO and do not create instances of any shared base class.  The specific `getPluginName` tests test the various scenarios around POJO's / string trees / etc.